lib/gitter-adapter.js

```diff
@@ -306,6 +307,19 @@ Adapter.prototype.listUsers = function(channel, key) {
   });
 };
 
+Adapter.prototype.autoJoin = function() {
+  this.gitterClient.currentUser()
+  .then(function(user) {
+    return user.rooms();
+  })
+  .then((function(rooms) {
+    rooms.forEach((function(room) {
+      if (room.oneToOne) return;
```

Comment:
This might be my incomplete understanding of Gitter but aren't one-to-one rooms basically private messages between users?